### ISSUE TITLE
SelectCheckboxMenu: disabled attribute with emptyLabel/label

### ISSUE THREAD

**Issue by dkralj** [primefaces/primefaces#9754]
### Describe the bug

When using disabled attirbute together with emptyLabel or label and when the component is disabled, label is not showing.
[primefaces-test.zip](https://github.com/primefaces/primefaces/files/10568615/primefaces-test.zip)

### PrimeFaces edition

Community

### PrimeFaces version

12.0.0

### JSF implementation

Mojarra

### JSF version

2.2

### Java version

1.8

**Comment by melloware** [comment]
cc @ltlBeBoy any interest in this one?

**Comment by ltlBeBoy** [comment]
Yes, you can assign to me. But it will take until next week...too much work.

**Comment by ltlBeBoy** [comment]
@melloware This issue was fixed by #9403. The console log shows the identical Javascript errors when running the reproducer. But with the current PrimeFaces version the label is rendered as expected.

**Comment by melloware** [comment]
Excellent. Thank you for confirming!

**Comment by ltlBeBoy** [comment]
@dkralj You can add the following MonkeyPatch to a Javascript file in your project (don't forget to clear the cache!) to fix that bug until PrimeFaces version 13.0.0 comes out:

```
if (PrimeFaces.widget.SelectCheckboxMenu) {
    PrimeFaces.widget.SelectCheckboxMenu.prototype.renderLabel = function() {
        if (!this.cfg.updateLabel) {
            return;
        }
        if (this.cfg.multiple) {
            this.multiItemContainer = this.multiItemContainer || this.jq.children('.ui-selectcheckboxmenu-multiple-container');
            var items = this.multiItemContainer.children();
            if (!items.length) {
                this.multiItemContainer.empty().append('<li class="ui-selectcheckboxmenu-emptylabel">' + (this.multiItemContainer.data('label') || '&nbsp;') + '</li>');
            }
        }
        else {
            // single select
            this.defaultLabel = this.label.text();
            this.label.css({
                'text-overflow': 'ellipsis',
                overflow: 'hidden'
            });
            this.updateLabel();
        }
    };
}

```

**Comment by dkralj** [comment]
@ltlBeBoy Thanks for the workaround. Cheers!
